brainstormforce/ultimate-addons-for-gutenberg
# Added filter for Spectra blocks to add a common data and remove example when `site-editor` is true

### Description

- Filter added for common data.
- Remove preview image when site-editor is true to resolve Style book issue.

### Screenshots

- NA

### Types of changes




- Filter for Pro
- Fix style book issue for FSE.

### How has this been tested?

- Go FSE theme.
- Open any Template 
- Verify preview of the block should not be visible in site editor.
- Go to page or post, click on + icon to add block.
- Hover on block Preview of block should be visible.

### Checklist:
